packages/core/src/checkout-buttons/strategies/braintree/braintree-paypal-button-strategy.ts

```diff
@@ -120,6 +142,8 @@ export default class BraintreePaypalButtonStrategy implements CheckoutButtonStra
     }
 
     private _renderPayPalMessages(messagingContainerId?: string): void {
+        if (!messagingContainerId) return;
```

Comment:
We already have this check in this method. It might be better to place state and cart variables inside IF block.